android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/login/LoginScreen.kt

```diff
@@ -23,17 +23,29 @@ import androidx.compose.ui.unit.dp
 import co.nimblehq.ic.kmm.suv.android.R
 import co.nimblehq.ic.kmm.suv.android.ui.components.PrimaryButton
 import co.nimblehq.ic.kmm.suv.android.ui.components.PrimaryTextField
+import co.nimblehq.ic.kmm.suv.domain.usecase.LogInUseCase
 import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.catch
+import org.koin.java.KoinJavaComponent.inject
 
 @Composable
-fun LoginScreen() {
+fun LoginScreen(
+    logInUseCase: LogInUseCase // TODO: Add this into ViewModel instead (Integrate task)
+) {
     var email by remember { mutableStateOf("") }
     var password by remember { mutableStateOf("") }
     var componentsVisible by remember {  mutableStateOf(false) }
 
     LaunchedEffect(Unit) {
         delay(4000)
         componentsVisible = true
+        logInUseCase("dev@nimblehq.co", "12345678")
+            .catch { error ->
+                print(error.message)
+            }
+            .collect { token ->
+                print(token)
```

Comment:
The default Kotlin print does not log anything into the Logcat for debugging. Let's use Android [Log](https://developer.android.com/reference/android/util/Log) or [Timber](https://github.com/nimblehq/android-templates/blob/74df13835f3aa4f296795eeb966f45a8c0be2007/template/app/src/main/java/co/nimblehq/template/ui/base/BaseNavigator.kt#L36) 🤓